armory/included/modules/Whois.py

```diff
@@ -108,13 +108,31 @@ def process_output(self, cmds):
         for cmd in cmds:
             if cmd["cidr"]:
                 _, cidr = self.ScopeCidr.find_or_create(cidr=cmd["cidr"])
-                cidr.meta["whois"] = open(cmd["output"]).read()
+                
+                ##need to handle potentially bad encoding with WHOIS data...
+                try:
+                    cidr.meta["whois"] = open(cmd["output"],encoding="utf-8").read()
+                except UnicodeDecodeError:
+                    pass
+                try: #is is latin??
+                    cidr.meta["whois"] = open(cmd["output"],encoding="latin-1").read()
```

Comment:
Could probably change the encoding to `iso-8859-1` instead of `latin-1` since it should handle most characters. Also since the code below is the same I would create a method which returns the text that way if any issue arises it will only need to be fixed in one place. Could place it within `armory.included.utilities` so it can be used in multiple modules.